### PR TITLE
Add environment tab (#586)

### PR DESCRIPTION
**Pull request by baev**
Not ready to merge

### CODE REVIEW

File: allure-report-face/src/main/webapp/plugins/environment/en.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "environment": {
+    "TITLE": "Environment",
+    "NO_CONTENT": "There is no environment variables"
```
Comment (pupssman) [inline allure-report-face/src/main/webapp/plugins/environment/en.json]: `There are no environment variables` OR `There is no environment defined`

**Comment by pupssman** [comment]
How  bout nice foldable tree view?

File: allure-report-face/src/main/webapp/plugins/environment/script.js [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+    var module = angular.module('allure.environment', []);
+
+    module.config(['$stateProvider', 'allureTabsProvider', function($stateProvider, allureTabsProvider) {
+        allureTabsProvider.addTab('environment', {title: 'environment.TITLE', icon: 'fa fa-cogs'});
+        allureTabsProvider.addTranslation('environment');
+    }]);
+    module.controller('EnvironmentCtrl', ['$scope', '$state', 'data', function($scope, $state, data) {
+        "use strict";
+
+        $scope.environment = data;
```
Comment (just-boris) [inline allure-report-face/src/main/webapp/plugins/environment/script.js]: obviously, we need some basic controller to make that stuff

**Comment by baev** [comment]
> How bout nice foldable tree view?

@pupssman you can implement it by yourself.

File: allure-report-face/src/main/webapp/plugins/environment/script.js (hunk 2 above)
Comment (baev) [inline allure-report-face/src/main/webapp/plugins/environment/script.js]: > obviously, we need some basic controller to make that stuff

could you please implement it?

**Comment by pupssman** [comment]
@baev could you please give a couple of pointers where to start?

**Comment by baev** [comment]
Run `mvn jetty:run -Pdev -f allure-report-face/`, then open `http://localhost:8080` to see the preview.

Then you can change whatever you want (your working directory should be `allure-report-face/src/main/webapp/plugins/environment`)

Preview will be updated after refresh without cache.

Data loaded to environment plugin - `allure-report-face/target/allure-report/data/environment.json`.

**Pull request closed without merging**